app/src/main/java/com/example/sharingang/items/ItemsViewModel.kt

```diff
@@ -24,28 +24,31 @@ class ItemsViewModel @Inject constructor(
     }
 
     enum class OBSERVABLES {
-        ALL_ITEMS, SEARCH_RESULTS, WISHLIST
+        ALL_ITEMS, SEARCH_RESULTS, USER_ITEMS, WISHLIST
     }
 
     private val _navigateToEditItem = MutableLiveData<Item?>()
-    val navigateToEditItem: LiveData<Item?>
+    private val navigateToEditItem: LiveData<Item?>
         get() = _navigateToEditItem
 
     private val _navigateToDetailItem = MutableLiveData<Item?>()
-    val navigateToDetailItem: LiveData<Item?>
+    private val navigateToDetailItem: LiveData<Item?>
         get() = _navigateToDetailItem
 
     private val _refreshing = MutableLiveData(false)
     val refreshing: LiveData<Boolean>
         get() = _refreshing
 
-
     private val _searchResults = MutableLiveData<List<Item>>(listOf())
     val searchResults: LiveData<List<Item>>
         get() = _searchResults
 
+    private val _userItems = MutableLiveData<List<Item>>()
+    private val userItems: LiveData<List<Item>>
+        get() = _userItems
+
     private val _wishlistItem : MutableLiveData<List<Item>> = MutableLiveData(ArrayList())
-    val wishlistItem : LiveData<List<Item>>
+    private val wishlistItem : LiveData<List<Item>>
```

Comment:
what's the point of making it private... the _wishlistItem is the private version of the variable this one is to be used for observers

Reply:
Yeah, I changed it because Android Studio complained, but I'll change it back.